packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -562,22 +556,22 @@ export default [
                 <CardBlock>
                   <div className='text-xs-center'>
                     <ActionButton
-                      active={state.snapshotMode}
+                      active={state.snapshotMode || undefined}
```

Comment:
What does it mean?

Reply:
`active` not works with the `false`  value.

Reply:
This is an issue in `Button`, please fix it instead.